internal/cli/inputs.go

```diff
@@ -26,15 +27,16 @@ func (i ProjectInputs) Filter() realm.AppFilter {
 
 // Resolve resolves the necessary inputs that remain unset after flags have been parsed
 func (i *ProjectInputs) Resolve(ui terminal.UI, wd string, skipAppPrompt bool) error {
-	app, _, appErr := local.FindApp(wd)
-	if appErr != nil {
-		return appErr
+	app, _, err := local.FindApp(wd)
+	if err != nil {
+		return err
 	}
 
 	if i.App == "" {
 		var appOption string
 
 		if app.RootDir != "" {
+			i.AppMeta = app.Meta
```

Comment:
`// set AppMeta when --app flag is not provided and FindApp locates an app directory`